_layouts/default.html

```diff
@@ -1,249 +1,185 @@
-                                    {% include anchor_headings.html html=content anchorClass='header-anchor' %}
-                                    {% endif %}
-                                </div>
-
-                                <div class="col-2 d-none d-lg-block">
-                                    <nav class="doc-toc">
-                                        {% include toc.html html=content h_max=2 item_class='toc-%level%' %}
-                                    </nav>
-                                </div>
-                            </div>
-                        </div>
-                    </main>
-                </div>
-            </div>
-
-            <footer class="page-footer">
-                <p class="page-footer-rights">© Swedbank Pay</p>
-            </footer>
-        </div>
-
-        <script src="{{ design_guide_version_url }}/scripts/dg.js"></script>
-        <script src="{{ '/assets/js/swedbank-pay-design-guide-theme.js' | relative_url }}"></script>
-        {% if site.google_analytics.tracking_id %}
-        {% include google_analytics.html %}
-        {% endif %}
-    </body>
-    {% if jekyll.environment == "development" %}
-    <script>document.write('<script src="http://' + (location.host || 'localhost').split(':')[0] + ':35729/livereload.js?snipver=1"></' + 'script>')</script>
-    {% endif %}
-
-</html>
\ No newline at end of file
+{%- assign title = page.title | default: "PayEx Design Guide theme for Jekyll" -%}
+{%- assign description = page.opengraph.description | default: site.opengraph.description -%}
+{%- assign image = page.opengraph.image | default: site.opengraph.image -%}
+{%- assign github_branch = site.github.branch | default : "develop" -%}
+{%- assign design_guide_base_url = site.design_guide.base_url | default: 'https://design.swedbankpay.com' -%}
+{%- assign design_guide_version = site.design_guide.version | default: '4.7.0' -%}
+{%- assign design_guide_version_url = design_guide_base_url | append: '/v/' | append: design_guide_version -%}
+{%- if page.sidebar.navigation == nil -%}
+    {%- unless page.name contains "index"  -%}
+        {%- assign dirname = page.dir | split: "/" | last | remove: "/" | capitalize -%}
+        {%- unless dirname == "" -%}
+            {%- assign title = dirname | append: " – " | append: title -%}
+        {%- endunless -%}
+    {%- endunless -%}
+{%- endif -%}
+<!DOCTYPE html>
+<html lang="{{ site.lang | default: "en-US" }}">
+    <head>
+        <meta charset="utf-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <meta name="mobile-web-app-capable" content="yes">
+        <meta name="theme-color" content="#000">
+        <meta name="application-name" content="{{ page.title }}">
+        <meta name="apple-mobile-web-app-capable" content="yes">
+        <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">
+        <meta name="apple-mobile-web-app-title" content="{{ page.title }}">
```

Comment:
Cant these lines also be moved into a include for apple icons?

Reply:
Sure.